arima_density.ipynb

```diff
@@ -0,0 +1,1771 @@
+{
+ "cells": [
```

Comment:
Lol had to open this file elsewhere: I don't know exactly what y'all are doing but maybe more descriptive variable names + some cell comments would be great